### DISCUSSION TITLE
How can I suppress the "Authors" heading in Quarto output?

### DISCUSSION THREAD

**Discussion by bateyt** [quarto-dev/quarto-cli#767] [Q&A] [answered]
Hello.

I've searched through the Quarto documentation and gallery, but have not been able to find an answer to my question.  How can I suppress the "Authors" heading on the output files that are rendered?  By default, it places the "Authors" heading under the page title, even if no authors are listed in the quarto.yml or index.yml files.  I can manually delete it in the index.html file, but it will reappear when rendered again.  It seems like there is probably a simple solution, but I'm at a loss.  Thanks in advance.

--Trey

**Comment by dragonstyle** [comment, marked as answer]
Can you confirm what version of Quarto you're using? We had this bug (the empty authors label appearing) and I think that we fixed it, so it could just be a version issue. If you're on the latest version, that means I probably somehow regressed it... You can check version that generated the page by viewing the source and looking for the meta generator tag...

**Reply by bateyt** [reply]
At the terminal, `quarto --version` returns 0.9.248.  Thanks!

**Reply by dragonstyle** [reply]
Can you try updating to the latest version? This was fixed recently enough that I think it will be resolved by an update. Thanks!

**Reply by bateyt** [reply]
That solved it.  Thank you!
